### DISCUSSION TITLE
GraphQL query with table field column filter

### DISCUSSION THREAD

**Discussion by yingban** [craftcms/cms#15088] [Ideas]
HI,

in Craft 4.x, it was not yet possible to query entries via GraphQL with a criteria on from table field.
Is it possible now with Craft 5?
For example, querying entries with the criteria table->col1 < 10.

Thank you!

**Comment by brandonkelly** [comment]
No, sorry. Recategorized this discussion as an idea, and we can treat it as a feature request.
